nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkService.java

```diff
@@ -47,6 +47,17 @@
      */
     String submit(BulkCommand command);
 
+    /**
+     * Submits a {@link BulkCommand} that will be processed asynchronously only if the transaction commits
+     * successfully (nothing will be submitted in case of transaction rollback).
+     * Note that the {@link #getStatus(Serializable)} will return an unknown state until transaction is committed.
+     *
+     * @since 2021.18
```

Comment:
up to 2021.19
2 more in BulkServiceImpl

Reply:
yes as commented in the 2021 branch the @since could be wrong because releases are promoted and it cannot always be known in advance, this will be updated only if there is another fix to push.